### DISCUSSION TITLE
Pricing

### DISCUSSION THREAD

**Discussion by ThomasDV1984** [opengisch/QField#3976] [Q&A]
1. I have upgraded to an organisation. In the past (beta testing) I had two accounts that where linked to the organisation, those members are now seen as community members. So I now pay €32 per month for 1gb of storage?
2. when adding aditional storage to my organisation I was not able to add 1GB of storage although I had only used 1.6 GB and had to buy 2GB.

**Comment by suricactus** [comment]
1) yes. You pay for minimum of two users and any other active member of your organization. If noone is active, you still pay for two users. If three people are active, you pay for three users, or 48 in your case.
2) let us know which is the organization name that expereinced technical issue.

**Reply by ThomasDV1984** [reply]
Thanks
organization nam: Bearfox
